Jenkinsfile

```diff
@@ -5,6 +5,18 @@ import com.tenable.jenkins.common.*
 import com.tenable.jenkins.Constants
 import com.tenable.jenkins.builds.snyk.*
 import com.tenable.jenkins.builds.nexusiq.*
+    
+
+
+def addParameters(final script, def theParams) {
+    
+    theParams << script.booleanParam(defaultValue: false,
+        description: 'Puplish to PYPI!', name: 'PUPBLISH2PYPI')
+
+    theParams
+}
+PropertiesHelper.defaultCICDProperties(this, 0, this.&addParameters)
```

Comment:
We should disable the params that we don't need:

i.e. 

replace this, 0 

with

this, Constants.PROP_NOCONSEC | Constants.PROP_NOSCHEDULE